core/urls.py

```diff
@@ -20,6 +21,14 @@
         ResourceListByCategoryView.as_view(),
         name='resource_list_view'
         ),
+    path(
+        'About_Us/',views.about,
```

Comment:
Bad naming convention-
`About_Us/` should be converted to `about/`

Reply:
resolve